terraform_env/dev/main.tf

```diff
@@ -0,0 +1,53 @@
+terraform {
```

Comment:
I said on our call recently, but worth also exploring how this could be achieved with [workspaces](https://www.terraform.io/docs/language/state/workspaces.html) - particularly helpful to stop you needing to keep half a dozen environments synced if this grows more complex